packages/extension-ui/src/partials/Header.tsx

```diff
@@ -29,17 +29,17 @@ interface Props extends ThemeProps {
   smallMargin?: boolean;
   text?: React.ReactNode;
   withStepper?: boolean;
-  withGoToRoot?: boolean;
+  customGoTo?: () => void;
```

Comment:
Does default `onAction` function has a type? Maybe we could use the same type to indicate which exactly function we expect here?